dotcom-rendering/src/server/render.editionsCrossword.web.tsx

```diff
@@ -0,0 +1,31 @@
+import { isString } from '@guardian/libs';
+import { EditionsCrosswordPage } from '../components/EditionsCrosswordPage';
+import { generateScriptTags, getPathFromManifest } from '../lib/assets';
+import { renderToStringWithEmotion } from '../lib/emotion';
+import type { FEEditionsCrosswords } from '../types/editionsCrossword';
+import { htmlCrosswordPageTemplate } from './htmlCrosswordPageTemplate';
+
+interface Props {
+	editionsCrosswords: FEEditionsCrosswords;
+}
+
+export const renderCrosswordHtml = ({
+	editionsCrosswords,
+}: Props): { html: string; prefetchScripts: string[] } => {
+	const { html } = renderToStringWithEmotion(
+		<EditionsCrosswordPage editionsCrosswords={editionsCrosswords} />,
+	);
+
+	const prefetchScripts = [
+		getPathFromManifest('client.editionsCrossword', 'index.js'),
+	].filter(isString);
+
+	const scriptTags = generateScriptTags([...prefetchScripts]);
```

Comment:
There's probably no need to spread this into a new array?

```suggestion
	const scriptTags = generateScriptTags(prefetchScripts);
```